mysqli/mysqli.php

```diff
@@ -358,9 +358,9 @@ public function get_server_info(): string {}
     public function get_warnings(): mysqli_warning|false {}
 
     /**
-     * Initializes MySQLi and returns a resource for use with mysqli_real_connect()
+     * Initializes MySQLi object
      * @link https://php.net/manual/en/mysqli.init.php
-     * @return mysqli an object.
+     * @return void
```

Comment:
Could you please share your arguments behind `void`? php-src [has](https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.stub.php#L165) this hinted as `bool|null`

Reply:
My bad. The correct signature is [`?false`](https://www.php.net/manual/en/mysqli.init.php), but I guess it should be `bool|null` in here as well, correct?

Reply:
Yes, thanks.